Jenkinsfile

```diff
@@ -102,13 +102,19 @@ pipeline {
                     }
                     post {
                         always {
-                            recordIssues enabledForFailure: true,
-                                         aggregatingResults: true,
-                                         id: "analysis-centos7",
-                                         tools: [ clang(), cppCheck() ],
-                                         filters: [excludeFile('.*\\/_build\\.external-Linux\\/.*'),
-                                                   excludeFile('_build\\.external-Linux\\/.*')]
+                            node('lightweight') {
+                                recordIssues enabledForFailure: true,
+                                             aggregatingResults: true,
+                                             id: "analysis-centos7",
+                                             tools: [ cppCheck() ],
```

Comment:
I think you still want to specify a compiler here, like `gcc4()` if the build is using _gcc_.  That being missing might be what's causing the:

```
ID cppcheck is already used by another action: io.jenkins.plugins.analysis.core.model.ResultAction for CPPCheck
```

error that you were getting.  Just a guess mind you.